run-experiment.sh

```diff
@@ -108,36 +127,36 @@ do
 				echo "HTTP Server CRIU Handler"
 				dump_criu_app
 
+				echo "Running bpftrace probes"
+				run_bpftrace
+
+				echo "Running execute requests script"
 				set +e
-				scale=0.1 image_path=$IMAGE_PATH ./$EXP_APP_NAME $HTTP_SERVER_ADDRESS / $REP_REQ $i $APP_DIR $HANDLER_TYPE $TRACE $APP_DIR/$CRIU_APP_OUTPUT >> $RESULTS_FILENAME
+				scale=0.1 image_path=$IMAGE_PATH ./$EXP_APP_NAME $HTTP_SERVER_ADDRESS / $REP_REQ $i $APP_DIR $HANDLER_TYPE $APP_DIR/$CRIU_APP_OUTPUT >> $RESULTS_FILENAME
 				EXECUTION_SUCCESS=$?
 
 				echo "Try to kill HTTP Server Handler process"
 				pgrep java
 				killall -v java
 				set -e
 
+				if [ $EXECUTION_SUCCESS -eq 0 ];
```

Comment:
Fazer algo se a execução for falha? Por exemplo, para e emitir uma mensagem clara o que aconteceu?

Reply:
Só a execução com o CRIU pode ter uma falha esperada (por causa da alocação dos PIDs dos processos) dessa forma o sucesso da execução do CRIU Server é não-determinística.

Eu posso esclarecer mais no erro no shell script imprimindo o código de saída do comando. No código que o script shell chama, o [execute-requests.go](https://github.com/paulofelipefeitosa/serverless-handlers/blob/integrate-tracers/server-http-handler/execute-requests.go), eu garanti que caso algo der errado ele emita uma mensagem na saída de erro que representa ou esclarece a falha.